coregrind/m_execontext.c

```diff
@@ -521,6 +521,12 @@ ExeContext* VG_(null_ExeContext) (void)
    return null_ExeContext;
 }
 
+void VG_(apply_ExeContext)( void(*action)(UInt n, Addr ip, void *uu_opaque),
```

Comment:
Please verify we really need this inside coregrind.
I think it can be implemented in pmemcheck using:
```
VG_(apply_StackTrace)( action, NULL, get_ExeContext_StackTrace(ec), n_ips );
```